src/SFA.DAS.EmployerDemand.Application/Demand/Commands/CreateCachedProviderInterest/CreateProviderInterestCommandValidator.cs

```diff
@@ -11,9 +11,9 @@ public Task<ValidationResult> ValidateAsync(CreateCachedProviderInterestCommand
         {
             var result = new ValidationResult();
 
-            if (item.EmployerDemandIds == null || !item.EmployerDemandIds.Any())
+            if (!item.EmployerDemands.Any())
             {
-                result.AddError(nameof(item.EmployerDemandIds), "Select the employers you're interested in");
+                result.AddError(nameof(item.EmployerDemands), "Select the employers you're interested in");
```

Comment:
You'll need to add a mapping, or change the view as otherwise the inline error messages wont show with this change
`ViewData.ModelState.ContainsKey("EmployerDemandIds")` probably should be changed to
`ViewData.ModelState.ContainsKey("EmployerDemands")` or use the name from the model using nameof